app/parse.py

```diff
@@ -8,8 +15,60 @@
     tags: list[str]
 
 
+QUOTE_FIELDS = ["text", "author", "tags"]
+
+HOME_PAGE = "https://quotes.toscrape.com/"
+
+
+def parse_single_quote(quote_element: Tag) -> [Quote]:
+    return Quote(
+        text=quote_element.select_one(".text").text,
+        author=quote_element.select_one(".author").text,
+        tags=[tag.text for tag in quote_element.select(".tag")],
+    )
+
+
+def get_next_page(page_soup:  BeautifulSoup) -> str | None:
+    next_url = page_soup.select_one(".pager .next a")
+    if next_url:
+        return next_url["href"]
+
+    return None
+
+
+def get_page_quotes(url: str = HOME_PAGE) -> [Quote]:
```

Comment:
The return type annotation for `get_page_quotes` should be `List[Quote]` instead of `[Quote]` to correctly indicate that it returns a list of `Quote` objects.